### PR TITLE
Change viewport styling to more closely match v4

### PR DESCRIPTION
**Pull request by connor-baer**
Issue: #5965

## What I did

A regression in #7385 re-added the subjectively ugly black border that had been removed in #5979 to address #5965. This PR reverts the border to its v4-like thinness.

## How to test

- Is this testable with Jest or Chromatic screenshots? this addon doesn't seem to have tests
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by connor-baer** [comment]
This PR is still relevant and waiting for a review :)

File: addons/viewport/src/Tool.tsx [hunk 1]
```diff
@@ -196,10 +196,10 @@ export const ViewportTool: FunctionComponent = memo(
             <Global
               styles={{
                 [`#${iframeId}`]: {
-                  margin: `auto`,
+                  margin: `${theme.layoutMargin}px auto`,
```
Comment (ndelangen) [inline addons/viewport/src/Tool.tsx]: I think this breaks being able to view the entire iframe when the iframe is bigger then then preview area:

Screenshot 2019-11-15 at 01 48 48

File: addons/viewport/src/Tool.tsx [hunk 2]
```diff
@@ -196,10 +196,10 @@ export const ViewportTool: FunctionComponent = memo(
...
                   transition: 'width .3s, height .3s',
                   position: 'relative',
-                  border: `${theme.layoutMargin}px solid black`,
+                  border: `1px solid black`,
                   borderRadius: theme.appBorderRadius,
```
Comment (ndelangen) [inline addons/viewport/src/Tool.tsx]: A single px border + 10px borderRadius together will mean we'll crop parts of the user's iframe.. I didn't want to do that, hence the border.

The user will not be able to click on the corners of their component, when using this addon.

**Review by ndelangen — CHANGES_REQUESTED** [review]

**Comment by ndelangen** [comment]
Does anyone else want to pitch in on this?

File: addons/viewport/src/Tool.tsx (hunk 1 above)
Comment (connor-baer) [reply addons/viewport/src/Tool.tsx]: I'll revert that change 👍

File: addons/viewport/src/Tool.tsx (hunk 2 above)
Comment (connor-baer) [reply addons/viewport/src/Tool.tsx]: Fair point that I hadn't considered 👍 What if we simply drop the border-radius?

Comment (ndelangen) [reply addons/viewport/src/Tool.tsx]: That'd be a design decision I'd like @domyen's opinion on

**Comment by connor-baer** [comment]
@domyen What do you think about this problem? https://github.com/storybookjs/storybook/pull/8486#discussion_r346618925

**Comment by domyen** [comment]
@connor-baer that sounds good to me! (was on holiday)

Thanks for your attention to detail 🙇‍♂️

**Comment by connor-baer** [comment]
Aaaaaah, sorry for requesting everyone's review. I merged `next` into my branch locally, I didn't really it added _all_ commits separately to the branch. I'll try to revert it...

**Review by ndelangen — APPROVED** [review]

**Review by domyen — APPROVED** [review]

**Comment by domyen** [comment]
lgtm thank you!

**Pull request merged**